src/Makefile.in

```diff
@@ -258,6 +258,10 @@ install: all
 	@install -m 0755 $(DRIVER_BIN) $(INSTALL_DIR)
 	@install -m 0755 portbind $(INSTALL_DIR)
 
+googletest: 
+	$(CXX) $(CXXFLAGS) $(LDFLAGS) -lgtest vm/internal/otable_test.cc -o otable_test
```

Comment:
Please add "@" ahead of each line 
Also,  Why don't you have to compile otable.o  and link with it?  The current way will have no way of setting up correct test coverage, because the source code is not otable.c.

Reply:
what is the difference between @-,-@,@? I am unsure which one to add.